process/testing/winfv/setup-fv.sh

```diff
@@ -17,7 +17,7 @@
 NAME_PREFIX="${NAME_PREFIX:=${USER}-win-fv}"
 
 # Kubernetes version
-KUBE_VERSION="${KUBE_VERSION:=1.20.1}"
+KUBE_VERSION="${KUBE_VERSION:=1.28.7}"
```

Comment:
We may have too many k8s (default) versions defined in multiple places for UT/FVs. Is it possible to use `K8S_VERSION` in [metadata.mk](https://github.com/projectcalico/calico/blob/master/metadata.mk#L11)?

Reply:
Good point, let me see how best to import that (Makefile format -> bash script)